plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -845,7 +847,7 @@ private String toRemoteSchemaName(String schemaName)
         return schemaName;
     }
 
-    private String toRemoteTableName(String schemaName, String tableName)
+    public String toRemoteTableName(String schemaName, String tableName)
```

Comment:
We don't want to expose `toRemoteSchemaName` & `toRemoteTableName` to the metadata class because the methods don't have cache. The right change seems adding a remote name to table handle class.

Reply:
I don't see how we can add it to `MongoTableHandle` - it is constructed in several places outside of `MongoSession`. I can add it to `MongoTable` easily, though.